disdrodb/tests/test_check_metadata.py

```diff
@@ -1,14 +1,249 @@
 import os
 import pytest
 import yaml
+import random
 
 
-from disdrodb.l0 import check_metadata
+from disdrodb.l0 import check_metadata, metadata

```

Comment:
Import here the functions to be tested rather than the module please (also in the other tests when you have some time). Make more readable/and short the tests

```
from disdrob.l0.check_metadata import (
check_metadata_geolocation,
identify_empty_metadata_keys,
# ....
)
```